mxcubeweb/core/components/queue.py

```diff
@@ -2077,6 +2079,21 @@ def init_queue_settings(self):
         self.app.AUTO_ADD_DIFFPLAN = HWR.beamline.collect.get_property(
             "auto_add_diff_plan", False
         )
+        # Change value of the parameter, without changing the hardware object property.
+        # This allows to properly reset the value on logout when invoking init_queue_settings.
+        self.app.REMEMBER_PARAMETERS_BETWEEN_SAMPLES = (
+            HWR.beamline.queue_manager.get_property(
+                "remember_parameters_between_samples", False
+            )
+        )
+
+        default_centring_method_as_string = HWR.beamline.queue_manager.get_property(
```

Comment:
I guess the same thing here then ?

Reply:
I meant for the naming with the `default` prefix :)

Reply:
Oh, changed then :)

Reply:
:)